src/groovy/com/unifina/api/CreateProductCommand.groovy

```diff
@@ -23,6 +23,6 @@ class CreateProductCommand {
 	Long minimumSubscriptionInSeconds = 0
 
 	static constraints = {
-		importFrom(Product)
+		importFrom(Product, exclude: ["name"])
```

Comment:
If the command object defines a default name, then can this `exclude` be removed?

If the `exclude` is needed, then is it needed in `SaveCanvasCommand` too?

Reply:
@hpihkala Incoming JSON string can contain en empty name. Should Command objects do just validation or also other tasks?